src/modules/land_detector/LandDetector.h

```diff
@@ -153,14 +159,27 @@ class LandDetector : public ModuleBase<LandDetector>, ModuleParams, px4::Schedul
 	systemlib::Hysteresis _ground_contact_hysteresis{true};
 	systemlib::Hysteresis _ground_effect_hysteresis{false};
 
-	actuator_armed_s _actuator_armed {};
+	actuator_armed_s         _actuator_armed{};
+	vehicle_acceleration_s   _vehicle_acceleration{};
+	vehicle_land_detected_s  _land_detected{};
+	vehicle_local_position_s _vehicle_local_position{};
 
-private:
+	orb_advert_t _land_detected_pub{nullptr};
 
-	void _check_params(bool force = false);
+	uORB::Subscription _actuator_armed_sub{ORB_ID(actuator_armed)};
+	uORB::Subscription _vehicle_acceleration_sub{ORB_ID(vehicle_acceleration)};
+	uORB::Subscription _vehicle_local_position_sub{ORB_ID(vehicle_local_position)};
+
+private:
 
 	void Run() override;
 
+	/**
+	 * Updates parameters if changes have occurred or if forced update.
+	 * @param force Force a parameter update.
+	 */
+	void _update_params(const bool force);
```

Comment:
```suggestion
	void _update_params(const bool force = false);
```
Instead of overloading with no parameter?